core/page-actions/page-actions-ui/langs/translation-en.json

```
@@ -1,8 +1,24 @@
 {
   "page.action.category.page.management.title": "Manage",
   "page.action.action.delete.page.title": "Delete",
+  "page.action.action.delete.page.cancel": "Cancel",
   "page.action.action.delete.page.dialog.title": "Delete Page",
   "page.action.action.delete.page.success": "The page \"{page}\" has been deleted.",
   "page.action.action.delete.page.error": "Failed to delete the page \"{page}\". Reason: [{reason}]",
-  "page.action.action.delete.page.confirm": "This operation will delete the page \"{page}\". Are you sure?"
+  "page.action.action.delete.page.confirm": "This operation will delete the page \"{page}\". Are you sure?",
+  "page.action.action.move.page.title": "Move",
+  "page.action.action.move.page.cancel": "Cancel",
+  "page.action.action.move.page.dialog.title": "Move Page",
+  "page.action.action.move.page.alert.content": "The page {pageName} already exists.",
+  "page.action.action.move.page.preserve.children.label": "Preserve Children",
+  "page.action.action.move.page.preserve.children.help": "Preserve the child pages by updating their paths and moving them to the new location.",
+  "page.action.action.move.page.success": "The page \"{page}\" has been moved to \"{newPage}\".",
+  "page.action.action.move.page.error": "Failed to move the page \"{page}\". Reason: [{reason}]",
+  "page.action.action.rename.page.title": "Rename",
+  "page.action.action.rename.page.cancel": "Cancel",
+  "page.action.action.rename.page.dialog.title": "Rename Page",
+  "page.action.action.rename.page.alert.content": "The page {pageName} already exists.",
+  "page.action.action.rename.page.name.label": "Name",
+  "page.action.action.rename.page.success": "The page \"{page}\" has been renamed to \"{newPage}\".",
+  "page.action.action.rename.page.error": "Failed to rename the page \"{page}\". Reason: [{reason}]"
```

Same here, I would the superfluous "the". wdyt?
```suggestion
  "page.action.action.move.page.success": "Page \"{page}\" has been moved to \"{newPage}\".",
  "page.action.action.move.page.error": "Failed to move page \"{page}\". Reason: [{reason}]",
  "page.action.action.rename.page.title": "Rename",
  "page.action.action.rename.page.cancel": "Cancel",
  "page.action.action.rename.page.dialog.title": "Rename Page",
  "page.action.action.rename.page.alert.content": "Page {pageName} already exists.",
  "page.action.action.rename.page.name.label": "Name",
  "page.action.action.rename.page.success": "Page \"{page}\" has been renamed to \"{newPage}\".",
  "page.action.action.rename.page.error": "Failed to rename page \"{page}\". Reason: [{reason}]"
```